src/dosbox.cpp

```diff
@@ -597,19 +597,23 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("nosound",Property::Changeable::OnlyAtStart,false);
 	Pbool->Set_help("Enable silent mode, sound is still emulated though.");
 
-	Pint = secprop->Add_int("rate",Property::Changeable::OnlyAtStart,44100);
+	Pint = secprop->Add_int("rate", Property::Changeable::OnlyAtStart, 44100);
 	Pint->Set_values(rates);
 	Pint->Set_help("Mixer sample rate, setting any device's rate higher than this will probably lower their sound quality.");
 
-	const char *blocksizes[] = {
-		 "1024", "2048", "4096", "8192", "512", "256", 0};
-	Pint = secprop->Add_int("blocksize",Property::Changeable::OnlyAtStart,1024);
+	const char *blocksizes[] = {"1024", "2048", "4096", "8192",
+	                            "512",  "256",  "128",  0};
+	Pint = secprop->Add_int("blocksize", deprecated, 1024);
 	Pint->Set_values(blocksizes);
-	Pint->Set_help("Mixer block size, larger blocks might help sound stuttering but sound will also be more lagged.");
+	Pint->Set_help("This property is deprecated, use latency instead.");
 
-	Pint = secprop->Add_int("prebuffer",Property::Changeable::OnlyAtStart,25);
+	Pint = secprop->Add_int("prebuffer", deprecated, 25);
 	Pint->SetMinMax(0,100);
-	Pint->Set_help("How many milliseconds of data to keep on top of the blocksize.");
+	Pint->Set_help("This property is deprecated, use latency instead.");
+
+	Pint = secprop->Add_int("latency", Property::Changeable::OnlyAtStart, 20);
+	Pint->SetMinMax(1, 100);
+	Pint->Set_help("Desired audio latency in milliseconds. The range is 1-100.");
```

Comment:
"desired" implies a goal, but with worse being possible/acceptable.
In this case, I think the maximum possible latency is what the user sets.. the buffer won't be bigger (and they will get stuttering/dropouts if their system is too slow to achieve this latency). 

My comment reads somewhat negative: but I think this this setting can be worded with stronger guarantees. 

Maybe maximum allowed latency? Note, stutter will occur if your PC cannot keep up. 
(That way, users will know "If I don't hear stuttering, then I'm good.. and they can keep going lower).

Reply:
We could add wording around the power of 2 limitation. Given that, we could have the latencies be a list instead of a range, since e.g. everything greater than ~10 (512 samples) but less than ~20 (1024) will still give you ~20 (1024).

Reply:
Yup, definitely like the locked down sizes like 2, 5, 10, 20, and know that I'm actually going to get roughly those values (if I stick with 48Khz). 

I would probably feel things aren't right id I set 11ms, but then latency blows out into the 20ms range (because of the underlying power of 2 rule). Just cut me off and only give me 10ms if that's the case 👍